examples/google-sheet/.gitignore

```diff
@@ -0,0 +1,132 @@
+# Extra files
```

Comment:
Would it make sense to use a terser gitignore? https://github.com/mui/mui-toolpad/blob/master/examples/with-prisma/.gitignore